sklearn/metrics/_regression.py

```diff
@@ -1700,12 +1721,12 @@ def d2_pinball_score(
     )
 
     if sample_weight is None:
-        y_quantile = np.tile(
-            np.percentile(y_true, q=alpha * 100, axis=0), (len(y_true), 1)
+        y_quantile = xp.tile(
+            xp.percentile(y_true, q=alpha * 100, axis=0), (len(y_true), 1)
```

Comment:
I don't find the `percentile` function in the documentation of the Array-API standard.